curriculum/challenges/english/25-front-end-development/workshop-fruit-search-app/67ecfbfe17c7d24465e0de88.md

```diff
@@ -0,0 +1,125 @@
+---
+id: 67ecfbfe17c7d24465e0de88
+title: Step 4
+challengeType: 0
+dashedName: step-4
+---
+
+# --description--
+
+Inside the `form` element, add an `input` element with an `id` of `search-input` and `type` `text`.
+
+# --hints--
+
+You should have an `input` element inside the `form` element with an `id` of `search-input`.
+
+```js
+const form = document.querySelector("form");
+assert.exists(form);
+
+const input = form.querySelector("input");
+assert.exists(input);
+assert.strictEqual(input.tagName, "INPUT");
+assert.strictEqual(input.getAttribute("id"), "search-input");
+```
+
+The `input` element should have a `type` value of `text`
```

Comment:
same here. search input is more appropriate here.
that should be change in all places throughout the workshop.